web/src/bootstrap_typeahead.ts

```diff
@@ -261,6 +261,9 @@ export class Typeahead<ItemType extends string | object> {
     // don't set the html content of the div from this module, and
     // it's handled from the caller (or updater function) instead.
     updateElementContent: boolean;
+    // Used to determine whether the typeahead should be shown,
+    // when the user clicks anywhere on the input element.
+    showOnClick: boolean;
```

Comment:
`ecf557e` in the commit message is not a full commit ID. Always use full commit IDs in commit descriptions; GitHub will shrink it appropriately for display.